env/helpers.bash

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+HELPERS_SCRIPT_DIR="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+HELPERS_LOG_FILE="/tmp/alkOS_build.log"
+
+source "${HELPERS_SCRIPT_DIR}/pretty_print.bash"
+
+dump_error() {
+  help
+
+  pretty_error "$1"
+  exit 1
+}
+
+assert_always() {
+  dump_error "ASSERT failed in ${BASH_SOURCE[1]} at line ${BASH_LINENO[0]}"
```

Comment:
How is this different from dump_error? Maybe merge the two

Reply:
Dump error is simple exiting function with some log message and maybe be used in all cases, assert fails when something unexpected happens, something what should never happen during execution.